android/examples/run-local-test/local.conf.js

```diff
@@ -32,7 +32,7 @@ exports.config = {
       'browserstack',
       { 
         browserstackLocal: true, opts: { forcelocal: false },
-        app: process.env.BROWSERSTACK_APP_ID || 'bs://<hashed app-id>'
+        app: './examples/LocalSample.apk' || 'bs://<hashed app-id>'
```

Comment:
here `||` is redundant. We may keep `process.env.BROWSERSTACK_APP_PATH || ./examples/LocalSample.apk`